README.md

```diff
@@ -144,7 +156,29 @@ Since the resource state is stored outside of the container, the GPU cluster lif
 10. ***[`OPTIONAL - GCS bucket not provided`]*** 
     > Need storage object owner access if you don't already have a storage bucket to reuse.
 
+
+## Billing Reports
+
+You can view your billing reports for your HPC cluster on the
+[Cloud Billing Reports](https://cloud.google.com/billing/docs/how-to/reports)
+page. ​​To view the Cloud Billing reports for your Cloud Billing account,
+including viewing the cost information for all of the Cloud projects that are
+linked to the account, you need a role that includes the
+`billing.accounts.getSpendingInformation` permission on your Cloud Billing
+account.
+
+To view the Cloud Billing reports for your Cloud Billing account:
+
+1. In the Google Cloud Console, go to `Navigation Menu` >
+   [`Billing`](https://console.cloud.google.com/billing/overview).
+2. At the prompt, choose the Cloud Billing account for which you'd like to view
+   reports. The Billing Overview page opens for the selected billing account.
+3. In the Billing navigation menu, select `Reports`.
+
+In the right side, expand the Filters view and then filter by label, specifying the key `aiinfra-cluster` and the desired value.
```

Comment:
I am not seeing the label filter. Is my billing account just too new or something?
![filters_menu](https://user-images.githubusercontent.com/25994930/219486383-e13a60d6-5ad4-496f-9334-d51b3132c661.jpg)

Reply:
yes. Once you create more resources you will see your billing account like this
![image](https://user-images.githubusercontent.com/112522451/219487045-f2d65643-279f-4f7b-8a4f-9430bb19182e.png)